### PR TITLE
test(e2e): migrate documentation tests to jest

### PR DESCRIPTION
**Pull request by gcusnieux**
**Issue**

https://github.com/gravitee-io/issues/issues/7606

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/7606-migrate-doc-test-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-e2e/jest.config.js [hunk 1]
```diff
@@ -172,11 +172,11 @@ module.exports = {
 
   // A map from regular expressions to paths to transformers
   transform: {
-    '^.+\\.ts$': 'esbuild-jest',
+    '\\.(js|jsx|ts|tsx)$': '@sucrase/jest-plugin',
     '^.+\\.xml$': '<rootDir>/lib/jest-raw-loader.js',
   },
 
-  // testTimeout: 5000,
+  testTimeout: 30000,
```
Comment (Okhelifi) [inline gravitee-apim-e2e/jest.config.js:179]: Is there a reason for this change ? 
If yes do we need to add it in the ci config too ?

**Review by Okhelifi — APPROVED** [review]

**Comment by Okhelifi** [comment]
I approved but I just seen that the CI jest e2e step failed 🙈

File: gravitee-apim-e2e/jest.config.js (hunk 1 above)
Comment (gcusnieux) [reply gravitee-apim-e2e/jest.config.js:179]: Yes, it's usefull on debug

**Comment by gcusnieux** [comment]
> I approved but I just seen that the CI jest e2e step failed 🙈

Yes, It's since docker improvements... @gaetanmaisse do you have an idea ?

**Pull request merged**